packages/website/src/About.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const About = () => (
+  <div>
+    <h2>About Page</h2>
+  </div>
+)
```

Comment:
Don't forget to add newlines at the end of the files - run the linter to make sure your code matches style guides